### PR TITLE
implemented support for testplan custom fields values obtaining

### PR DESCRIPTION
**Pull request by maiksaray**
Implemented support for testplan custom fields

This was needed to integrate jenkins with testlink in a more elegant way

### CODE REVIEW

**Comment by kinow** [comment]
Just confirmed in 1.9.14 that the method getTestPlanCustomFieldDesignValue is available in the XML-RPC API.

**Comment by kinow** [comment]
There are a few parameters missing. The getTestPlanCustomFieldDesignValue method in the PHP API calls the getCustomFieldValue internal method.

This last method receives: 

- ~~scopeParamName~~ ("scope") (set by the getTestPlanCustomFieldDesignValue with 'design' value)
- testPlanIDParamName ("testplanid")
- testProjectIDParamName ("testprojectid")
- customFieldNameParamName ("customfieldname")
- ~~nodeTypeParamName~~ ("nodetype") (set by the getTestPlanCustomFieldDesignValue with 'testplan' value)
- nodeIDParamName ("nodeid")
- executionIDParamName ("executionid")
- linkIDParamName ("linkid")

But if the method works without the other parameters, let's add it anyway and we will add the remaining parameters another time.

File: src/main/java/br/eti/kinoshita/testlinkjavaapi/model/TestPlan.java [hunk 1]
```diff
@@ -24,6 +24,8 @@
 package br.eti.kinoshita.testlinkjavaapi.model;
 
 import java.io.Serializable;
```
Comment (kinow) [inline src/main/java/br/eti/kinoshita/testlinkjavaapi/model/TestPlan.java]: Excellent work! Will bump the serialVersionUID too as the TestPlan constructor and properties changed. Kudos!

**Comment by kinow** [comment]
Looking good! Merging.

**Comment by maiksaray** [comment]
Thanks for the attitude! Are you going to update version in mvn repo soon?

**Comment by kinow** [comment]
Scheduled for Sunday noon New Zealand time - https://github.com/kinow/testlink-java-api/blob/b94c530259de75e79f353c9080f72cbd58268b3f/src/changes/changes.xml#L8

Should happen on Sunday, or in case I don't have spare time on Sunday, definitely Monday evening.

**Comment by maiksaray** [comment]
Oh, many thanks to you, sorry for not finding it on my own

**Comment by kinow** [comment]
Not a problem, just committed that moments ago :smiley:

**Pull request merged**